_data/developer.yml

```diff
@@ -164,6 +164,12 @@ websites:
       tfa: Yes
       software: Yes
       doc: https://context.io/security-faq
+      
```

Comment:
a lot of whitespace there. Could you remove it?

Reply:
Sure thing.